### ISSUE TITLE
Dataloader next(batch_gen['train'])

### ISSUE THREAD

**Issue by santiago-pires** [bsmarine/BleedDetection#10]
Hi Brett, 
Thanks for sharing the code! I am studying medical images and I am trying to use your implementation. Actually I am getting an error that I cannot discover the reason and you have commented something about it in the code. When trying to pick a new batch it is stalled. This is what you commented in lines around 125 of exec.py.
##Stalled
batch = next(batch_gen['train']) ######## Instead of this line, grab a batch from training data fold
I would appreciate your help, 
All the best,
Santiago

**Comment by bsmarine** [comment]
Hi Santiago, thanks for reaching out. What type of data are you attempting to train on? Eg modality, postprocessed shape, etc

The comments in the code refer to planned changes. Not sure why its stalled in your case. Could be an issue with the batch generator.

-Brett

**Comment by santiago-pires** [comment]
Hi Brett, thanks for you answer!
I was able to fix that, it was an error that I was doing but it was exactly the same as commented. 
I am training 'retina-unet' now, but I am observing only the part of the loss function of the segmentation decreasing. The part of classification and regression are practically constant. I was wondering if you have used 'retina-unet' or just 'mrcnn' as stated in the code. Great work modifying the nms function by the way!

**Comment by bsmarine** [comment]
Glad you were able to figure out! I've primarily been using as FRCNN for object detection.

I can't take credit for the NMS modifications. My modifications were only to extend to multiphase CT (4D) vs singlephase CT/MRI (3D). If you haven't already please see the latest repo by the original authors from DKFZ for semantic segmentation. https://github.com/MIC-DKFZ/nnUNet
